### PR TITLE
Update de-DE.ini

### PR DESCRIPTION
**Pull request by MarcoWilli**
Added (most) missing translations

### CODE REVIEW

File: data/obs-plugins/obs-kinect/locale/de-DE.ini [hunk 1]
```diff
@@ -94,8 +94,8 @@ ObsKinectV1.NearMode="Aktiviere Depth Near-Modus"
 ObsKinectV1.NearModeDesc="Reduziert den Tiefenfokus auf 40-200 cm anstatt 80-400 cm (funktioniert nicht für Kinect 360 Version)"
 
 ; obs-kinect-sdk20 backend
-ObsKinectV2.NexusLedIntensity="Nexus led intensity" ; missing translation
-ObsKinectV2.PrivacyLedIntensity="Privacy led intensity" ; missing translation
+ObsKinectV2.NexusLedIntensity="Nexus LED Intensität"
+ObsKinectV2.PrivacyLedIntensity="Privacy LED Intensität" ; missing translation - what´s -privacy led- ?
```
Comment (SirLynix) [inline data/obs-plugins/obs-kinect/locale/de-DE.ini:98]: Privacy LED is the "activation" LED, the small one on the left which indicates the Kinect camera is on.

File: data/obs-plugins/obs-kinect/locale/de-DE.ini [hunk 2]
```diff
@@ -39,52 +39,52 @@ ObsKinect.GreenScreenMaxDirtyDepthDesc="Reduziert Flackern zu Lasten der Bewegun
...
+ObsKinect.Gain_Red="Verstärkung rot"
+ObsKinect.Gamma="Gamma" ;
+ObsKinect.Hue="Farbton" ;
+ObsKinect.PowerlineFrequency="Flimmerreduzierung (Netzfrequenz)"
+ObsKinect.PowerlineFrequency_Disabled="Deaktiviert"
+ObsKinect.PowerlineFrequency_50Hz="50 Hz"
+ObsKinect.PowerlineFrequency_60Hz="60 Hz"
+ObsKinect.Saturation="Sättigung"
+ObsKinect.Sharpness="Schärfe"
+ObsKinect.WhiteBalance="Weißabgleich" ; don´t see this one in gui
```
Comment (SirLynix) [inline data/obs-plugins/obs-kinect/locale/de-DE.ini:83]: This is for "White balance value", in kelvin. It's a setting of Kinect v1 and Azure Kinect.

File: data/obs-plugins/obs-kinect/locale/de-DE.ini [hunk 3]
```diff
@@ -39,52 +39,52 @@ ObsKinect.GreenScreenMaxDirtyDepthDesc="Reduziert Flackern zu Lasten der Bewegun
...
-ObsKinect.ExposureMode="Exposure mode" ; missing translation
-ObsKinect.ExposureTime="Belichtungszeit" ; not sure
+ObsKinect.Contrast="Kontrast"
+ObsKinect.DigitalGain="Digitale Verstärkung"
+ObsKinect.DumpCameraSettings="Dump camera settings" ; missing translation - what does it do?
+ObsKinect.ExposureCompensation="Belichtungsausgleich"
+ObsKinect.ExposureControl_FullyAuto="Automatisch"
+ObsKinect.ExposureControl_SemiAuto="Halbautomatisch"
+ObsKinect.ExposureControl_Manual="Manuell"
+ObsKinect.ExposureIntegrationTime="Exposure integration time" ; missing translation - what does it do?
```
Comment (SirLynix) [inline data/obs-plugins/obs-kinect/locale/de-DE.ini:67]: I'm not sure what's exactly the difference with ExposureTime, but it's quite similar. Kinectv2 parameters are not documented.

File: data/obs-plugins/obs-kinect/locale/de-DE.ini [hunk 4]
```diff
@@ -39,52 +39,52 @@ ObsKinect.GreenScreenMaxDirtyDepthDesc="Reduziert Flackern zu Lasten der Bewegun
...
-ObsKinect.ExposureCompensation="Exposure compensation" ; missing translation
-ObsKinect.ExposureControl_FullyAuto="Fully automatic" ; missing translation
-ObsKinect.ExposureControl_SemiAuto="Semi automatic" ; missing translation
-ObsKinect.ExposureControl_Manual="Manual" ; missing translation
-ObsKinect.ExposureIntegrationTime="Exposure integration time" ; missing translation
-ObsKinect.ExposureMode="Exposure mode" ; missing translation
-ObsKinect.ExposureTime="Belichtungszeit" ; not sure
+ObsKinect.Contrast="Kontrast"
+ObsKinect.DigitalGain="Digitale Verstärkung"
+ObsKinect.DumpCameraSettings="Dump camera settings" ; missing translation - what does it do?
```
Comment (SirLynix) [inline data/obs-plugins/obs-kinect/locale/de-DE.ini:62]: This writes your current camera settings to obs logs (which may be useful to see which values are taken in automatic mode)

File: data/obs-plugins/obs-kinect/locale/de-DE.ini (hunk 1 above)
Comment (MarcoWilli) [reply data/obs-plugins/obs-kinect/locale/de-DE.ini:98]: OK. Would translate it with "Status LED Intensität"
"Nexus" is unclear, too. Sorry for not marking it correctly.

File: data/obs-plugins/obs-kinect/locale/de-DE.ini (hunk 2 above)
Comment (MarcoWilli) [reply data/obs-plugins/obs-kinect/locale/de-DE.ini:83]: Translation is correct. Was just wondering about.

File: data/obs-plugins/obs-kinect/locale/de-DE.ini (hunk 1 above)
Comment (SirLynix) [reply data/obs-plugins/obs-kinect/locale/de-DE.ini:98]: No problem, the Nexus LED is the xbox logo one.

File: data/obs-plugins/obs-kinect/locale/de-DE.ini (hunk 3 above)
Comment (MarcoWilli) [reply data/obs-plugins/obs-kinect/locale/de-DE.ini:67]: I would translate it with "Belichtungsintegrationszeit". Typical german translation. Not that easy to translate such technical terms without knowing/seeing what they do.

File: data/obs-plugins/obs-kinect/locale/de-DE.ini (hunk 4 above)
Comment (MarcoWilli) [reply data/obs-plugins/obs-kinect/locale/de-DE.ini:62]: I know the term from a developers sight but I´m quite sure no private user knows this. Would translate it with "Kameraeinstellungen in Log schreiben".

File: data/obs-plugins/obs-kinect/locale/de-DE.ini (hunk 1 above)
Comment (MarcoWilli) [reply data/obs-plugins/obs-kinect/locale/de-DE.ini:98]: I see. Then I think "Xbox LED Helligkeit" and "Status LED Helligkeit" (helligkeit = brightness) would be more precise/understandable.

**Pull request closed without merging**